### PR TITLE
Add DOI

### PR DESCRIPTION
**Pull request by mlazzarin**
I noticed that I didn't add the DOI of the published paper anywhere, so I replaced the arXiv identifier in the citation policy (README.md) with the DOI. Should I keep both of them?
By the way, the documentation is full of references linked to the arXiv page. Should I replace them with the DOI or it is just fine to keep as it is, so that readers can access the paper for free?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,5 +15,5 @@ MCNNTUNES documentation is available at [mcnntunes.readthedocs.io](https://mcnnt
 ## Citation policy
 
 If you use the package please cite the following references:
-- https://arxiv.org/abs/2010.02213
```
Comment (scarrazza) [inline README.md:18]: Let me suggest to keep both arxiv and doi.

**Comment by scarrazza** [comment]
@mlazzarin thanks for this, I think you can merge #20 here before merging.

**Review by scarrazza — APPROVED** [review]
Perfect, thanks.

**Comment by mlazzarin** [comment]
Regarding the links to the paper in the documentation, should I place the DOI everywhere or it is fine to keep the arxiv link?

**Comment by scarrazza** [comment]
> Regarding the links to the paper in the documentation, should I place the DOI everywhere or it is fine to keep the arxiv link?

I think the arxiv number should be enough.

**Comment by mlazzarin** [comment]
Ok, thanks.

**Pull request merged**